torch/__init__.py

```diff
@@ -1708,5 +1708,18 @@ def registerOp(cls, op_key, full_schema, op_impl, dispatch_key):
             return cls.ops_table[(op_key, dispatch_key)]
 
 
+# Deprecated attributes
+_deprecated_attrs = {
+    "has_mps": torch.backends.mps.is_built,
+    "has_cuda": torch.backends.cuda.is_built,
+    "has_mkldnn": torch.backends.mkldnn.is_available,
+}
+def __getattr__(name):
+    v = _deprecated_attrs.get(name)
```

Comment:
```suggestion
    replacement = _deprecated_attrs.get(name)
```
Nit, but could we name v something more semantically close to what it represents?